src/libcore/option.rs

```diff
@@ -1107,3 +1107,26 @@ impl<A, V: FromIterator<A>> FromIterator<Option<A>> for Option<V> {
         }
     }
 }
+
+/// The `Option` type. See [the module level documentation](index.html) for more.
+#[unstable(feature = "try_trait", issue = "42327")]
+#[derive(Clone, Copy, PartialEq, PartialOrd, Eq, Ord, Debug, Hash)]
+pub struct Missing;
```

Comment:
Yes, I needed something there in order to compile and forgot to change it. I'm not sure what to replace it with; maybe `The equivalent of Option::None for a Result type.`?